lib/zoho_hub/base_record.rb

```diff
@@ -241,4 +250,6 @@ def build_response(body)
       self.class.build_response(body)
     end
   end
+
+  require 'zoho_hub/modules/note'
```

Comment:
@Fabien-Wecasa I moved the code into its own file but i do need to require the module after the ZohoHub::BaseRecord is defined

Reply:
Sorry, misunderstanding: the related functions `notes` and `add_note` should also be moved in the module, and I guess you won't need this require anymore. (see the link in my previous comment)

Reply:
https://github.com/Fabien-Wecasa/zoho_hub/pull/5/files#diff-c28dcd616422d5d722bb0cb44e786ba5dc1064e1d52cc2ab489e22cd3ec98f13